demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -334,28 +335,28 @@ def copy_packs_content_to_packs_bundle(self, packs):
                 os.mkdir(dest_dir)
                 if dir_name in DIR_TO_PREFIX:
                     packages_dirs = get_child_directories(content_dir)
-                    for package_dir in packages_dirs:
-                        ymls, _ = get_yml_paths_in_dir(package_dir, error_msg='')
-                        if not ymls or (len(ymls) == 1 and ymls[0].endswith('_unified.yml')):
-                            msg = 'Skipping package: {} -'.format(package_dir)
-                            if not ymls:
-                                print_warning('{} No yml files found in the package directory'.format(msg))
-                            else:
-                                print_warning('{} Only unified yml found in the package directory'.format(msg))
-                            continue
-                        package_dir_name = os.path.basename(package_dir)
-                        unifier = Unifier(package_dir, dir_name, dest_dir)
-                        unifier.merge_script_package_to_yml()
-
-                        # also copy CHANGELOG markdown files over (should only be one per package)
```

Comment:
you don't need the changelog anymore?

Reply:
We are switching to global changelog soon